src/layout/layout_entity_flipbook.cpp

```
@@ -1,13 +1,40 @@
+                        std::string defaultValue;
+                        if (target == AnimationTrack::Target::FlipbookPlaying) {
+                            defaultValue = "0";
+                        }
+                        it->second.GetOrCreateKeyframe(0) = std::move(defaultValue);
+                    }
+                }
+            }
+        }
+    }
+
     LayoutEntityFlipbook::LayoutEntityFlipbook(LayoutRect const& initialBounds)
```

Clang (14.0.6)

[error] constructor does not initialize these fields: m_flipbookPath

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)